devel/libuv/Portfile

```diff
@@ -56,3 +56,33 @@ depends_build       port:automake \
                     port:autoconf \
                     port:libtool \
                     port:pkgconfig
+
```

Comment:
please remove the extra <cr> here ... 2 is plenty, so 1 empty line between the prior and next text